lapce-ui/src/explorer.rs

```diff
@@ -686,6 +690,29 @@ impl Widget<LapceTabData> for FileExplorerFileList {
                             menu = menu.entry(item);
                         }
 
+                        menu = menu.separator();
+                        let path_to_file = node.path_buf.clone();
+                        let item = druid::MenuItem::new("Copy Absolute Path")
```

Comment:
VSCode calls this "Copy Path" and I also think "Absolute" is not really needed here.

Reply:
Sometimes it's necessary. I think that is good when the user has an variants and can choose.

Reply:
It should be:
`Copy path`
`Copy relative path`

Reply:
Sorry I meant the word isn't necessary, so the options should be "Copy Path" and "Copy Relative Path". This is what VSCode calls these, too.

Reply:
> Sorry I meant the word isn't necessary, so the options should be "Copy Path" and "Copy Relative Path". This is what VSCode calls these, too.

Oh, okay. I missunderstood, well, I do not insist in this case, I just like to be meticulous.
Anyway, if you think that would be better to remove this word, welp.

Reply:
I think we just meant the UI text, not internal code

Reply:
> I think we just meant the UI text, not internal code

Then the uniformity of the code is lost.